packages/components/src/entities/SampleHeader.tsx

```diff
@@ -0,0 +1,318 @@
+                                )}
+
+                                {!isMedia && (
+                                    <RequiresPermission user={user} perms={PermissionTypes.Insert}>
+                                        <AssayImportSubMenuItem
+                                            queryModel={sampleModel}
+                                            providerType={assayProviderType}
+                                            requireSelection={false}
+                                            disabled={!canUploadAssayData}
+                                        />
+                                    </RequiresPermission>
+                                )}
+
+                                {!isMedia && (
+                                    <RequiresPermission user={user} perms={PermissionTypes.ManagePicklists}>
+                                        <AddToPicklistMenuItem
+                                            queryModel={sampleModel}
+                                            sampleIds={[sampleId]}
+                                            user={user}
+                                        />
+                                        <PicklistCreationMenuItem
+                                            sampleIds={[sampleId]}
+                                            key="picklist"
+                                            user={user}
+                                        />
+                                    </RequiresPermission>
+                                )}
+                                <RequiresPermission user={user} perms={PermissionTypes.ManageSampleWorkflows}>
+                                    <DisableableMenuItem href={jobCreationHref} operationPermitted={canAddToWorkflow}>
+                                        Create Workflow Job
+                                    </DisableableMenuItem>
+                                </RequiresPermission>
+
+                                {!isMedia && !!StorageMenu && <StorageMenu onUpdate={onUpdate} sampleModel={sampleModel} />}
+
+                                {canPrintLabels && <MenuItem onClick={onPrintLabel}>Print Labels</MenuItem>}
+
+                                {!isMedia && (
+                                    <RequiresPermission user={user} perms={PermissionTypes.Delete}>
+                                        <DisableableMenuItem
+                                            disabledMessage={getSampleDeleteMessage(canDelete, error)}
+                                            onClick={onDeleteSample}
+                                            operationPermitted={canDelete}
+                                        >
+                                            Delete {entityDataType?.nounSingular ?? 'Sample'}
+                                        </DisableableMenuItem>
+                                    </RequiresPermission>
+                                )}
+                                {!isMedia && (
```

Comment:
I would assume that LKB might want to view audit history for media as well. Is that possible here?

Reply:
I waffled on whether to do this now or with the Media consistency story since it's not included now and doesn't show as much detail as for other samples (in my brief look). I'll add it back and see if anyone looks askance at it (or looks at it at all).